slither.config.json

```diff
@@ -1,6 +1,6 @@
 {
     "solc_remaps": "@=node_modules/@",
-    "filter_paths": "node_modules",
+    "filter_paths": "(node_modules/|contracts/modules/interfaces/|contracts/libs/|contracts/external/|contracts/registries/)",
```

Comment:
Does scanning `contracts/libs` & `contracts/registries` creates issues (because of solidity version)?

Reply:
It does not. We can add it, for the moment most of the alerts are related to pragma version / mixedCase so could be ignored in the slither.config.json with  `detectors_to_exclude`